publish helm chart on release (fixes #1114)

ikreymer opened the pull request:
- no longer using :latest by default in values.yaml, instead updating version with each release
- set chart version to match app version in Chart.yaml
- update version in helm chart and values.yaml as part of update-version.sh script
- update test.yaml and local-config.yaml to enable using :latest tag images
- ci: add ci script for packaging current helm chart
- bump to 1.6.4-beta.0 for release with helm chart

ikreymer committed:
publish helm chart on release (fixes #1114)
- no longer using :latest by default in values.yaml, instead updating version with each release
- set chart version to match app version in Chart.yaml
- update version in helm chart and values.yaml as part of update-version.sh script
- update test.yaml and local-config.yaml to enable using :latest tag images
- ci: add ci script for packaging current helm chart
- bump to 1.6.4-beta.0 for release with helm chart

tw4l commented on .github/workflows/publish-helm-chart.yaml:
```diff
@@ -0,0 +1,37 @@
+      - name: Package Chart
+        run: |
+          mkdir .chart-out
+          helm package chart/ --destination .chart-out
```
Will this include `local.yaml` if it's been copied here, e.g. if someone is using `btrix`?

tw4l replied:
Same question for `examples/` I suppose - I don't seen an option in `helm package` to exclude certain files

ikreymer replied:
yes, will include examples. this is designed to run only on CI so shouldn't have a `local.yaml` there

ikreymer committed:
docs:
- update docs to indicate deploying from GitHub release
- automatically fill in latest version for 'VERSION' using custom script

ikreymer committed:
remove extra tags

ikreymer committed:
make 'local_service_port: 30870' default in values.yaml, only use if no ingress

ikreymer committed:
fix test
